Sources/AsyncHTTPClient/HTTPHandler.swift

```diff
@@ -1025,8 +1019,7 @@ extension TaskHandler: ChannelDuplexHandler {
                 /// Some HTTP Servers can 'forget' to respond with CloseNotify when client is closing connection,
                 /// this could lead to incomplete SSL shutdown. But since request is already processed, we can ignore this error.
                 break
-            case .head where self.ignoreUncleanSSLShutdown,
-                 .body where self.ignoreUncleanSSLShutdown:
+            case .sending where self.ignoreUncleanSSLShutdown:
```

Comment:
test `testNoResponseWithIgnoreErrorForSSLUncleanShutdown` fails if I add `.sent`, I'm not sure why yet...

Reply:
.sending means that we have not sent client `.end`

Reply:
What's the difference between `.sent` and `.endOrError`? In many cases (`content-length` present) after having sent the response body, we're done?

Reply:
you are looking at slightly outdated code

Reply:
this is a slightly weird part, it tries to do the following: if `ignoreUncleanSSLShutdown` is set and we received response `.head`, then it's allowed to not fail